### DISCUSSION TITLE
Add a new category

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#733] [Q&A] [answered]
### Contact Details

District6wc@gmail.com

### Requested Feature/Enhancment

Can I get some clarification on how to handle the XYZ here?

if (function_exists('tsml_custom_types')) {
		tsml_custom_types(array(
			'XYZ' => 'My Custom Type',
		));
	}

1. Be careful with the codes ("XYZ" in the above example) as this gives you the ability to replace existing types.
Should I just create a random 3 letter sequence for each new category?

Thank you
Rose L

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

**Comment by tech2serve** [comment, marked as answer]
"XYZ" should be whatever code (usually an abbreviation) you want to use for your custom type: "My Custom Type". While most users won't ever see this, you probably want it to be meaningful to you.

For example, let's say you want to add a custom type for meetings that serve special cake, you might set the key as "CAKE" and the description as "Special Cake Meeting".

Here are the [current accepted types](https://github.com/code4recovery/spec#meeting-types) for the spec used by Meeting Guide, but if I remember right you're Al-Anon so it shouldn't matter too much if you want to overwrite them. 

BTW, I'm glad you're coming here for help, but please don't open issues. As the directions say, please open a Discussion when you need something. If the maintainers decide a bug is present, or they validate a new feature, we'll open an issue for it.

Thanks.

**Reply by district6wc** [reply]
Thank you for clarifying this for me. I appreciate the thoroughness of this explanation. And I will open a discussion next time if I need something

**Comment by district6wc** [comment]
Hi

I am hoping to follow up on this as it was a long while ago, We have a new
parent-focused group and that is a category on the WSO site but I don't see
it on your list. DO you have an updated code for Parents?


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328
